rdrf/rdrf/helpers/utils.py

```diff
@@ -990,3 +990,25 @@ def supports_deidentification_workflow():
     if settings.DEIDENTIFIED_SITE_ID:
         return True
     return False
+
+
+def same_working_group(patient, user):
```

Comment:
add a registry model argument and test member of a shared working group in that registry